frontend/src/components/GoalCards/GoalCard.scss

```diff
@@ -18,29 +19,83 @@ $max-width-date-column: 50%;
   flex-basis: clamp($min-width-text-column, 100%, 100%);
 }
 
+.ttahub-goal-card__goal-column:not(:last-child) {
+  margin-bottom: 16px;
+}
+
+.ttahub-goal-card__goal-topics-tool-tip {
+  display :none;
+}
+
+.ttahub-goal-card__goal-topics-csv {
+  display: block;
+}
+
+@media(min-width: 627px) {
+  .ttahub-goal-card__goal-column:not(:nth-child(n+3):nth-child(-n+5)) {
+    margin-bottom: 16px;
+  }
+
+  .ttahub-goal-card__goal-topics-csv {
+    display: block;
+  }
+
+  .ttahub-goal-card__goal-topics-tool-tip {
+    display: none;
+  }
+}
```

Comment:
You don't need to write these CSS so many times/at each breakpoint. You just need the default state, outside of a media query (hide the tooltip, show the CSV) and then the reverse at the minimum width where you'd want it to flip.